### DISCUSSION TITLE
wavelink.Node() is not iterable

### DISCUSSION THREAD

**Discussion by ahryoo** [EvieePy/Wavelink#296] [Q&A] [answered]
With wavelink v3 and going from NodePool to Pool, nodes are required to be iterable.
However, wavelink.Node() is not iterable and wavelink.Pool still requires a list.

```python
node = wavelink.Node(
    identifier=None,
    uri='http://192.168.178.xxx:2333',
    password='xxx'
)

await wavelink.Pool.connect(
    nodes=[node],
    client=self.bot
)

```

**Comment by chillymosh** [comment]
@ahryoo What is the point you are trying to get at? You can have multiple Node objects in the list assigned to nodes.

e.g. 
```py
node = wavelink.Node(
    uri='http://192.168.178.xxx:2333',
    password='xxx'
)
node2 = wavelink.Node(
    uri='http://some.url.com:2333',
    password='xxx'
)

await wavelink.Pool.connect(
    nodes=[node, node2],
    client=self.bot
)
```

I don't understand why you think a Node object would be iterable?

**Reply by ahryoo** [reply]
If I'm not mistaken, it is in the docs:

`classmethod await connect(*, nodes: Iterable[Node]...`

https://wavelink.dev/en/latest/wavelink.html#pool

**Reply by chillymosh** [reply, marked as answer]
That annotation means that nodes is an iterarable that contains Nodes, not Node is an iterable

**Reply by ahryoo** [reply]
I guess it's time to take a break.
